libs/command/context.go

```diff
@@ -0,0 +1,13 @@
+package command
```

Comment:
Did you consider other package names? This one is not quite descriptive.

Reply:
Had perplexity come up with a few suggestions. I like `cmdctx`, will update the package name to that.

Reply:
Why is command / cmd relevant here? This seems to be related to databricks sdk setup.

Reply:
Addressed in https://github.com/databricks/cli/pull/2462

Reply:
> Why is command / cmd relevant here? 

Because these values are set once and are global for the entire command run, like the workspace client, account client and exec id.

We could migrate other globals like loggers to this package to consolidate those as well and then there's only one package that gets or sets values in the context. Not a priority for me though.